examples/main/main.cpp

```diff
@@ -347,9 +347,10 @@ int main(int argc, char ** argv) {
             set_console_color(con_st, CONSOLE_COLOR_DEFAULT);
         }
 
-        // in interactive mode, and not currently processing queued inputs;
+        // in not currently processing queued inputs;
         // check if we should prompt the user for more
-        if (params.interactive && (int) embd_inp.size() <= n_consumed) {
+        // or quit
```

Comment:
You can shorten these three comments to just:
```C++
        // if not currently processing queued inputs
```

Reply:
Done.